pkg/scheduler/scheduler.go

```diff
@@ -460,9 +460,13 @@ func (s *Scheduler) QuerierLoop(querier schedulerpb.SchedulerForQuerier_QuerierL
 
 		r := req.(*schedulerRequest)
 
-		s.queueDuration.Observe(time.Since(r.enqueueTime).Seconds())
+		reqEnqueueTime := time.Since(r.enqueueTime).Seconds()
+		s.queueDuration.Observe(reqEnqueueTime)
 		r.queueSpan.Finish()
 
+		level.Info(s.log).Log("msg", "querier request dequeued", "queryID", r.queryID,
```

Comment:
Could be interesting to add the querier id too so that we can see if they dequeue fairly across all of them.

Reply:
Q: Wondering if the original plan is to bring this queue information as a part of logging in [metrics.go](https://github.com/grafana/loki/blob/main/pkg/logql/metrics.go#L96-L108)?

https://github.com/grafana/loki/issues/4774

Reply:
@kavirajk adding this information as part of logging in metrics.go makes sense, however, we already have this information as part of the scheduler request. To add this information as part of metrics, we would need to in some way pass it downstream to the logql metrics. I wasn't able to do that in a simpler way, but let me know if I missed something 🙂

Reply:
I really like this idea! is it possible to get the tenant ID here as well?

Reply:
@slim-bean good idea. Added in commit https://github.com/grafana/loki/pull/4949/commits/242176181d6504a69d5f4f878a0e1973e97df552
From what I understood by looking at the [enqueueRequest method](https://github.com/grafana/loki/blob/a5c5f1693857594adb4f59e3176fe941d5939bd9/pkg/scheduler/scheduler.go#L366), a query can be multi-tenant. Therefore, used the method `tenant.TenantIDsFromOrgID` by using the scheduled request's orgId (`r.userID`). 

However, it is not clear the distinction between tenant and orgId in this context. What is the difference between tenant and org?